source/features/comments-time-machine-links.tsx

```diff
@@ -65,12 +65,18 @@ async function showTimemachineBar(): Promise<void | false> {
 		url.pathname = pathnameParts.join('/');
 	} else {
 		// This feature only makes sense if the URL points to a non-permalink
-		const branchSelector = await elementReady('.branch-select-menu i');
+		const branchSelector = await elementReady('[data-hotkey="w"] i');
 		const isPermalink = /Tag|Tree/.test(branchSelector!.textContent!);
 		if (isPermalink) {
 			return false;
 		}
 
+		// "Repository refresh" layout uses the itemprop="dateModified"
+		const lastCommitDate = await elementReady('.repository-content .Box.Box--condensed relative-time, [itemprop="dateModified"] relative-time');
```

Comment:
Should we do it like this to keep the comment near the actual code?

```suggestion
		const lastCommitDate = await elementReady([
			'.repository-content .Box.Box--condensed relative-time',
			'[itemprop="dateModified"] relative-time' // "Repository refresh" beta
		].join(','));
```